docs/release-instructions.md

```diff
@@ -1,11 +1,16 @@
 # Steps for creating a new release
 
-1. Create release branch (branch name can be the target version number).
-2. Update version number in `emmopy/__init__.py`.
-3. Commit and push to origin ([emmo-repo/EMMO-python](https://github.com/emmo-repo/EMMO-python.git)).
-4. Push the release branch to GitHub and create a pull request for merging to the master branch.
-5. Once accepted, merge to master.
```

Comment:
Why delete this part of the description?

Reply:
Because it's no longer relevant to making a release. All of these things will be done in the GH Actions workflow.

Reply:
good, thanx.

Reply:
Let's maybe go through the workflows together?